cardano_node_tests/utils/tx_view.py

```diff
@@ -145,4 +178,8 @@ def check_tx_view(  # noqa: C901
     if tx_raw_len_certs != loaded_len_certs:
         raise AssertionError(f"certificates: {tx_raw_len_certs} != {loaded_len_certs}")
 
+    # check collateral inputs
+    if not _check_collateral_inputs(tx_raw_output, tx_loaded["collateral inputs"]):
```

Comment:
Use `tx_loaded.get("collateral inputs")` as "collateral inputs" is not present for Tx eras older than Alonzo.